src/nickelmenu.cc

```diff
@@ -231,6 +232,33 @@ extern "C" __attribute__((visibility("default"))) MenuTextItem* _nm_menu_hook(vo
     return AbstractNickelMenuController_createMenuTextItem(_this, menu, label, checkable, checked, thingy);
 }
 
+QString nm_menu_pixmap(const char *custom, const char *custom_temp_out, const char *fallback) {
+    if (!custom || !custom_temp_out)
+        return QString(fallback);
+
+    QPixmap a;
+    if (!a.load(QString(custom))) {
```

Comment:
I'll change this to a QImage, which should fix the crash for large images.

Reply:
Might be a few days until I have time to test this again (we're, uh, using the winter holidays to repaint the bathroom :D).